### PR TITLE
[PowerMock]LPS 151641

### PR DESCRIPTION
**Pull request by JanisZhang**

### CODE REVIEW

**Comment by JanisZhang** [comment]
Hi @tinatian @dantewang,

This pull is to remove PowerMock in modules/apps/dynamic-data-mapping-data-provider-instance.

https://issues.liferay.com/browse/LPS-151641

Thanks for checking.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-instance/src/test/java/com/liferay/dynamic/data/mapping/data/provider/instance/internal/WorkflowDefinitionsDataProviderTest.java [hunk 1]
```diff
@@ -170,52 +171,41 @@ public void testThrowDDMDataProviderException() throws Exception {
...
-
-		when(
-			LocaleUtil.toLanguageId(_locale)
-		).thenReturn(
-			"en_US"
-		);
-	}
-
 	private void _setUpWorkflowDefinition(
 		WorkflowDefinition workflowDefinition1, String name, String title) {
```
Comment (dantewang) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-instance/src/test/java/com/liferay/dynamic/data/mapping/data/provider/instance/internal/WorkflowDefinitionsDataProviderTest.java:187]: @JanisZhang Could you append a separate commit to this pull to SF this parameter name `workflowDefinition1` -> `workflowDefinition`? It's strange that a common method has a parameter named `xx1`.

**Review by dantewang — APPROVED** [review]
Changes approved, an SF recommended.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-instance/src/test/java/com/liferay/dynamic/data/mapping/data/provider/instance/internal/WorkflowDefinitionsDataProviderTest.java (hunk 1 above)
Comment (JanisZhang) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-instance/src/test/java/com/liferay/dynamic/data/mapping/data/provider/instance/internal/WorkflowDefinitionsDataProviderTest.java]: @dantewang sure.

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina.

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/116288

**Pull request closed without merging**